.circleci/config.yml

```diff
@@ -123,7 +103,6 @@ jobs:
       - aws-ecr-setup
       - gcp-gcr/gcr-auth:
           registry-url: us-west1-docker.pkg.dev
-      - run: echo $GITHUB_CONTAINER_REGISTRY_TOKEN | docker login ghcr.io -u USERNAME --password-stdin
```

Comment:
do you need this back?

Reply:
nope. Just using GCP Artifact registry now for the caching - one dependency less. Also it appeared to me that the github container registry is a bit flaky at times.